### ISSUE TITLE
Tabs ADA Title II non-compliant in horizontal orientation

### ISSUE THREAD

**Issue by brianparmstrong** [twbs/bootstrap#42677]
### Prerequisites

- [x] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [x] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [x] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

Per WAI-ARIA [tabs pattern](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/), the up and down arrow keys should not be listened for in horizontal orientation.  The [Bootstrap example](https://getbootstrap.com/docs/5.3/components/navs-tabs/#javascript-behavior) shows it being listened for.  The minified bootstrap bundle shows it being listened for.  I am unable to override it in my own code, leaving the websites I work on ADA Title II non-compliant where Tabs are used.

I have verified that nothing in my own code is listening for the ArrowDown nor ArrowUp keydown event.

Issue occurs in all desktop browsers.

### Reduced test cases

See the [Bootstrap example](https://getbootstrap.com/docs/5.3/components/navs-tabs/#javascript-behavior).

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome

### What version of Bootstrap are you using?

v5.3.8

**Comment by sarathkumarsasi** [comment]
@julien-deramond  Can I work on this?
